.github/workflows/build-and-publish.yml

```diff
@@ -0,0 +1,42 @@
+name: Build and Publish to NPM
+on:
+    push:
+        branches: [master, development]
```

Comment:
```suggestion
        branches:
      - main
      - beta
      - next
      - '^([0-9]|[1-9][0-9]*)\.([0-9]|[1-9][0-9]*)\.([0-9]|[1-9][0-9]*|x)'
```

Reply:
How about this 🤔 
Base on the documentation we can define sth like this

Reply:
This will cause the workflow to only run on `main` branch so `master` should still be there, but I forgot to add `beta` and `next` branches here, nice catch 👍

Reply:
My bad 😁 

```
      - master
      - beta
      - next
      - '^([0-9]|[1-9][0-9]*)\.([0-9]|[1-9][0-9]*)\.([0-9]|[1-9][0-9]*|x)'
```